src/admin/getting_started_checks/design_removeDefaultBannersFrontPage.php

```diff
@@ -49,6 +49,6 @@ public function isComplete()
     {
         $name = 'front_page.tpl';
         $custom = file_get_contents(geoTemplate::getFilePath(geoTemplate::MAIN_PAGE, '', $name));
-        return (strpos($custom, '<a href="#"><img src="{external file=\'images/banners/banner2.jpg\'}" alt="" /></a>') === false);
+        return (strpos($custom, '<img src="{external file=\'images/banners/sample_300x100.jpg\'}"') === false);
```

Comment:
I think it might be best to check for the old or newer version of the file.  Someone could be using an older version of the templates and it would be good to detect the old style as well.

Reply:
Good point, but I should note that this didn't work anyway because the template did not match the checker script.